pkg/new-ui/v1beta1/frontend/src/app/pages/experiment-creation/trial-template/trial-template.component.html

```diff
@@ -37,7 +37,13 @@
     <mat-form-field appearance="outline" class="fit-content">
       <mat-label>Source Type</mat-label>
       <mat-select formControlName="type">
-        <mat-option value="configmap">ConfigMap</mat-option>
+        <mat-option
+          [disabled]="!templates.length"
+          [matTooltip]="!templates.length ? 'No TrialTemplates found' : ''"
```

Comment:
Let's keep it separate
```suggestion
          [matTooltip]="!templates.length ? 'No Trial templates found' : ''"
```

Reply:
Makes sense!